src/bridge/settings/openedx/mfe/slot_config/mitxonline/learning-mfe-config.env.jsx

```diff
@@ -1,11 +1,18 @@
-import { getConfig } from '@edx/frontend-platform';
 import config from './common-mfe-config.env.jsx';
-import * as remoteTutorDrawer from "./mitodl-smoot-design/dist/bundles/remoteTutorDrawer.umd.js"
 
-
-remoteTutorDrawer.init({
-  messageOrigin: getConfig().LMS_BASE_URL,
-  transformBody: messages => ({ message: messages[messages.length - 1].content }),
+import(
+  /**
+   * remoteTutorDrawer is already bundled to include its own version
+   * of React and ReactDOM.
+   *
+   * Add webpackIgnore to avoid bundling it again.
+   */
+  /* webpackIgnore: true */
```

Comment:
I don't entirely understand what was causing the error in https://github.com/mitodl/hq/issues/7259

However, previously:
1. We were bundling the code (including transpile/minification) via Vite during `smoot-design`'s build phase. *The purpose of this pre-bundling was largely to avoid conflicts with OpenEdx's version of React*)
2. The code was getting bundled (including minification/transpile) again via Webpack during the MFE build.

I noticed that replacing the synchronous import with an async import loading smoot via https://unpkg.com/@mitodl/smoot-design@6.4.0/dist/bundles/remoteTutorDrawer.es.js made the error go away entirely. This leads me to believe the double-bundling was causing problems.

So, lets not double bundle.

Reply:
BTW: We could use https://unpkg.com/@mitodl/smoot-design@6.4.0/dist/bundles/remoteTutorDrawer.es.js in production rather than copying and loading from the static folder.

However, I don't think this a great idea. Unpkg does work pretty well, but I don't think it's intended to be a production CDN.